talent/job-descriptions/senior-ds.md

```diff
@@ -0,0 +1,92 @@
+
+- [Readme files found in all GitHub repositories (16M, October 2016](https://data.world/vmarkovtsev/github-readme-files)
+
+- [≈ 452,000,000 commits' metadata taken from 16,000,000 repositories on GitHub (Oct 2016)](https://data.world/vmarkovtsev/452-m-commits-on-github)
+
+
+### ROLE
+
+- Strong computer science and machine learning background is essential for data science team members. You will be expected to be a passionate, skillful engineer who is able to produce amazing results quickly and reliably. Coding skills are important; we are using Python 3 and Go in our research and production prototyping. Besides, we occasionally code in C++ and CUDA. MLoSC shares common ideas with Natural Language Processing, so solid **NLP** knowledge is also required. **Deep Learning experience** is highly appreciated.
+
+
+## CULTURE
+
+- source{d} is a company for developers by developers. We firmly believe in always doing what's best for the community, not necessarily companies. "For developers" is the answer to any strategic question or dilemma in the company. Always For Developers.
+
+- "By developers" is our identity. Everyone in the company is either a developer or has a developer/engineering mindset. We don't hire anyone who doesn't fit into this culture, it's the only way to make sure everyone understands developers and can always defend their interest.
+
+- At the moment, we are 20+ people from 10 different countries working closely together from our office in Madrid, Spain. We are more than happy to sponsor you a visa and guide you and your family through the whole process if you decide to come to work from our office, but you may also choose to work remotely as part of our team does working from Portugal, Estonia, Russia, and others.
+
+- At source{d}, we have a transparent salary policy which we feel strongly about it. Although a seniority level for every candidate will be determined during the last round of on-site interviews, one can expect that for a Senior Engineer salary ranges from €60.000 to €70.000. You can find our net amounts [here](https://cincodias.elpais.com/herramientas/calculadora-sueldo-neto/) (in Spanish only).
+
+- Following the same level of transparency as with development process, all the above aspects of the work culture are documented and publicly available in our Guide at https://github.com/src-d/guide/.
+
+## PERKS 
+
+ - EU Visa support.
+ - Open Source Days, 10% of the time you can work on any OSS project you choose.
+ - Flexible hours and the possibility of remote work.
+ - We go to conferences and other developer events ! ;)
+ - Free books. Company will buy any books that can help you be even better at your work.
+ - Spacious physical office, close to public transit and to the Retiro park.
+ - Kitchen, full of supplies from beers to fresh fruits.
+ - Monthly get-togethers, annual summer and winter Xmas parties, a retreat.
```

Comment:
I would change "a retreat" to "hackathons" because few people outside of the company know about it means